spec/spec_helper.rb

```diff
@@ -8,14 +8,15 @@
 #
 # Details of default values for these configuration options can be seen at
 # https://github.com/colszowka/simplecov/blob/master/lib/simplecov/configuration.rb#L217
-#
-SimpleCov.minimum_coverage 90
-# SimpleCov.minimum_coverage_by_file 80
-# SimpleCov.maximum_coverage_drop 5
-# SimpleCov.refuse_coverage_drop
-SimpleCov.use_merging false
-
 SimpleCov.start do
```

Comment:
is there a reason we don't use the built in 'rails' config? this would give us a lot of the filtering up-front

Reply:
No idea 🤷

Reply:
TIL about Simplecov profiles (not sure how I've managed to miss them until now :shrug:). The rails profile is https://github.com/simplecov-ruby/simplecov/blob/main/lib/simplecov/profiles/rails.rb and seems useful.

```suggestion
SimpleCov.start("rails") do
```